test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -899,6 +899,14 @@ def test_snapshot_obf_stress(self):
                 self.assertIn(ret["out"], ["", None, []], str(val) +
                               " was not obfuscated. Real values:\n" +
                               str(real_values))
+                # Also make sure that no filenames contain the sensitive val
+                cmd = ["find", snap_dir, "-name",  "\'*" + str(val) + "*\'"]
+                ret = self.du.run_cmd(cmd=cmd, as_script=True,
```

Comment:
You shouldn't need the as_script.  All the arguments are part of find.  It isn't a big deal though.  It's just a little slower.